.circleci/config.yml

```diff
@@ -107,6 +119,25 @@ jobs:
         user: node
     <<: *unit_tests
 
+  publish_shared:
+    docker:
+      - image: node:8
+        user: node
+    steps:
+      - checkout
+      - run:
+          name: Switch to the common directory
+          command: cd src/shared
+      - run:
+          name: Install modules and dependencies.
+          command: npm install --unsafe-perm
```

Comment:
no need to use `--unsafe-perm` since we run as non-root account (here and everywhere)